### ISSUE TITLE
Support "terraform import"

### ISSUE THREAD

**Issue by okrause** [NetApp/terraform-provider-netapp-gcp#33]
Problem description:
I am trying to import an existing volume to put it under TF management. TF provides the "terraform import" feature for this.

Importing works like this:

1. Create an empty resource file
```tf
resource "netapp-gcp_volume" "myvolume" {
}

```
2. Find the ID of the volume to import (e.g. in UI)
3. Run the import
`terraform import netapp-gcp_volume.myvolume 1bc88bc6-cc7d-5fe3-3737-8e635fe2f996`
4. Use `terraform show` to update the empty resource file with the actual settings.
5. Use `terraform plan` to verify that the resource file against the deployed volume.

Problem:
Import fails:
```bash
terraform import netapp-gcp_volume.myvolume 1bc88bc6-cc7d-5fe3-3737-8e635fe2f996
netapp-gcp_volume.myvolume: Importing from ID "1bc88bc6-cc7d-5fe3-3737-8e635fe2f996"...
netapp-gcp_volume.myvolume: Import prepared!
  Prepared netapp-gcp_volume for import
netapp-gcp_volume.myvolume: Refreshing state... [id=1bc88bc6-cc7d-5fe3-3737-8e635fe2f996]

Error: <html><body><h1>403 Forbidden</h1>
Request forbidden by administrative rules.
</body></html>

```

Reason:
Import calls getVolumeByID in volume.go. Unfortunately, getVolumeByID needs two parameters. It required the volumeIU (=volume UUID), which is passed by terraform import and the region. AFAIK, there is no way to pass the region into terraform import.

Potential solutions:
1) Rewrite getVolumeByID in such a way that it will find the correct region. This would work very similar to how getVolumeByNameOrCreationToken works (very all volumes and filter for the right volume). Concern here is, that two volumes in different region might have the same ID. Very unlikely, but possible.
2) Write a proper importer. In resourceGCPVolume we currently do passthrough:
```go
		Importer: &schema.ResourceImporter{
			State: schema.ImportStatePassthrough,
		},
```
An importer build similar to getVolumeByNameOrCreationToken (but using VolumeId) could even return multiple volumes (see https://github.com/hashicorp/terraform/blob/master/helper/schema/resource_importer.go)

**Comment by okrause** [comment]
PS: I tested an import by hardcoding the region in getVolumeByID and my volume was properly imported.

**Comment by okrause** [comment]
FYI: I implemented a prototype using approach 1. It implements a findVolumeRegionByID function, which getVolumeByID calls if region is unspecified. If more than one volume for same ID is found, it errors out. So far, it works fine. If maintainers want to go down that solution path, I can create a PR. Just ping me.

**Comment by lonico** [comment]
as a third option, would it be possible to use an environment variable to indicate the region?

Though my preference would be for a proper importer.

My only real concern is that method 1 will fail if we see the same volume id in two regions.   How unlikely is it?

**Comment by okrause** [comment]
I agree, proper importer is the best solution.
Regarding volume ID collision: Since it is a UUID, it should be very, very unlikely. But someone who knows the backend details need to tell.

**Comment by wenjun666** [comment]
For either way, we need api that will return all regions?

**Comment by okrause** [comment]
> For either way, we need api that will return all regions?

Such an API would be very helpful for all kind of use cases.

We don't need such an API for this issue. It is sufficient to list all existing volumes (independent of region) and then filter for volumeID. Let me send you my changes as an PR. Not necessarily for inclusion, but as a proposal for discussion.

**Comment by wenjun666** [comment]
I think such api doesn't exist, but your workaround is great. we can get region by id in the importer function. I will go ahead create a ticket to include the functionality.

**Comment by okrause** [comment]
This was added in #347d04e0eca639b04a1537d68991fc36aceb1eea and is now available with v21.2.0.
